File: fahrenheit-celsius.cpp
#include<iostream>

using namespace std;

int main(int argc,  char *argv[]) {
	double fahrenheit;
	double celsius;

	cout << "Enter the temperature in fahrenheit: ";
	cin >> fahrenheit;

	celsius = (fahrenheit - 32) * 5/9;

	cout << "The temperature in °C " << celsius << endl;
	return 0;
}

File: fahrenheit-celsius.py
def convert_to_celsius(fahrenheit):
	return (fahrenheit - 32) * 5/9

def main():
	fahrenheit = input("Please enter temperature (°F): ")
	celsius = convert_to_celsius(float(fahrenheit))
	print("Equivalent temperature in °C " + str(celsius))

if __name__ == '__main__':
	main()